src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = number => {
+    if (number === undefined) {
+      const current = sum;
```

Comment:
I think a name like currentSum would be clearer

Reply:
agree